spec/unit/pops/evaluator/access_ops_spec.rb

```diff
@@ -24,115 +24,115 @@ def binary(s)
     # Note that the factory is not aware of Binary and cannot operate on a
     # literal binary. Instead, it must create a call to Binary.new() with the base64 encoded
     # string as an argument
-    CALL_NAMED(QREF("Binary"), true, [Base64.strict_encode64(s)])
+    CALL_NAMED(QREF("Binary"), true, [infer(Base64.strict_encode64(s))])
   end
 
   context 'The evaluator when operating on a String' do
     it 'can get a single character using a single key index to []' do
-      expect(evaluate(literal('abc')[1])).to eql('b')
+      expect(evaluate(literal('abc').test_access(1))).to eql('b')
```

Comment:
not too happy about the name `test_access` as I keep reading it as a verb rather than a noun. Can we rename it to something that makes more sense? (I.e. I read it as something that tests access, not that is a special kind of access that is only performed by tests.